.github/workflows/updatecli/updatecli.d/bump-latest-snapshot-version.yml

```
@@ -53,12 +53,34 @@ targets:
       matchpattern: '(STACK_VERSION:) 8\.[^\s]*\.[^\s]*'
       replacepattern: '$1 {{ source "latestSnapshot" }}'
 
-  update-snapshot-label:
-    name: '[updatecli] Update latest snapshot label step to {{ source "latestSnapshotMajorMinor" }}'
+  update-snapshot-label-daily:
+    name: '[updatecli] [daily] Update latest snapshot label step to {{ source "latestSnapshotMajorMinor" }}'
     kind: file
     sourceid: latestSnapshotMajorMinor
     scmid: default
     spec:
       file: '.buildkite/pipeline.schedule-daily.yml'
       matchpattern: '(Stack Version) v8\.\d+"'
       replacepattern: 'Stack Version v{{ source "latestSnapshotMajorMinor" }}"'
+
+  ## TODO: Looks like these targets are running with the contents from main branch
+  ## Uncomment this target once this code is merged
+  #update-snapshot-daily:
```

Maybe this conflicts with the existing targets and this is why they are not being launched? :thinking: 

```suggestion
  #update-snapshot-weekly:
```

---

Thanks!